zerver/actions/scheduled_messages.py

```diff
@@ -73,6 +75,7 @@ def do_schedule_messages(
     *,
     read_by_sender: bool = False,
     skip_events: bool = False,
+    delivery_type: int = ScheduledMessage.SEND_LATER,
```

Comment:
Let's not set a default here -- the caller should just pass this always.

Reply:
Done.